src/components/Faq/Faq.vue

```diff
@@ -0,0 +1,489 @@
+<template>
+  <div class="flex flex-col justify-center pt-20 pb-10 md:pt-28">
+    <p class="text-center text-4xl">Frequently Asked Questions</p>
```

Comment:
can u use the same font here?

like this font 
![image](https://user-images.githubusercontent.com/73097560/143485145-36c60948-343d-44b8-9d37-9a2215911722.png)

and tell me if you're done adding all the FAQ so we can merge

Reply:
Done 👍